clang/test/Driver/sycl-offload.c

```diff
@@ -69,6 +69,13 @@
 // RUN:   | FileCheck -check-prefix=CHK-SYCL-TARGET %s
 // CHK-SYCL-TARGET-NOT: error: SYCL target is invalid
 
+/// -fsycl-device-obj argument check
+// RUN: %clang -### -fsycl-device-obj=test -fsycl %s 2>&1 \
```

Comment:
What is the behavior when `--offload-new-driver` or `--no-offload-new-driver` is passed?

Reply:
The behavior is the same.  The use of `--offload-new-driver` vs `--no-offload-new-driver` does not change the need for the diagnostic or the behavior involved with `-fsycl-device-obj`.